templates/blog/topics/snappy.html

```diff
@@ -0,0 +1,46 @@
+{% extends "templates/one-column.html" %} 
+
+{% block title %}Snapcraft and Snaps{% endblock %}
+
+{% block meta_description %}Package any app for any Linux and deliver updates directly. All the latest from the Snappy team &mdash; articles on application architecture, prototyping, packaging paradigms and more.{% endblock %}
+
+{% block content %} 
+
+ <section class="p-strip--accent p-blog-topic__strip is-dark is-shallow u-image-position" style="overflow: hidden; background-color: #333">
+    <div class="row header-strip u-equal-height">
+      <div class="col-6">
+        <div class="p-heading-icon">
+          <div class="p-heading-icon__header">
+            <img class="p-blog-topic__icon" src="{{ASSET_SERVER_URL}}b46c8d7d-snapcraft-logo-hor-white.svg" alt=""/>
+          </div>
+          <h2 class="p-heading--two">Package any app for any Linux and deliver updates directly</h2>
```

Comment:
Can you make this a `p` with the heading two class. This is technically a strap line